release.yaml

```diff
@@ -10,8 +10,10 @@ fleet-crd:
 - 101.1.0+up0.6.0-rc.2
 neuvector:
 - 101.0.2+up2.4.0
+- 102.0.0+up2.4.2
 neuvector-crd:
 - 101.0.2+up2.4.0
+- 102.0.0+up2.4.2
```

Comment:
We need another commit to remove 101.0.2+up2.4.0 from neuvector and neuvector-crd chart

Reply:
Removed the unreleased verion from releae.yaml